src/expr/node_manager_template.cpp

```diff
@@ -849,18 +886,7 @@ TypeNode NodeManager::mkFunctionType(const std::vector<TypeNode>& argTypes,
 
 TypeNode NodeManager::mkTupleType(const std::vector<TypeNode>& types)
 {
-  std::vector<TypeNode> ts;
```

Comment:
The purpose of this vector is a mystery !!

Reply:
Haha yes, I know its origin, this is leftover from when the old API's `cvc4::Type` had to be converted to `cvc4::TypeNode`